### ISSUE TITLE
Revive Spotiflyer

### ISSUE THREAD

**Issue by s0mt0chukwu** [Shabinder/SpotiFlyer#3218]
Apparently it seems that Spotiflyer is dead 
I wish someone build a new model on top Spotiflyer apk

**Comment by Shabinder** [comment]
I am working on refactoring soundbound, to make it easy usable as was spotiflyer, and with that and some fixes already scheduled.

It is capable to do what spotiflyer could and even more than.

So for that stay tuned, shall be good to go within a week or two.

If something else is missing/bothering you lmk.

**Comment by s0mt0chukwu** [comment]
> I am working on refactoring soundbound, to make it easy usable as was spotiflyer, and with that and some fixes already scheduled.
> 
> It is capable to do what spotiflyer could and even more than.
> 
> So for that stay tuned, shall be good to go within a week or two.
> 
> If something else is missing/bothering you lmk.
> 

But SoundBound isn't open source??

**Comment by Shabinder** [comment]
yes soundbound isnt opensource yet, no plans to make it as of now

**Comment by IMrCreepyI** [comment]
will soundbound come to platforms other then mobile?

**Comment by Shabinder** [comment]
Yes it will.
